apps/amo/management/commands/clean_redis.py

```diff
@@ -99,21 +69,47 @@ def file_keys():
             total[1], round(float(total[1]) / total[0] * 100, 1)))
 
 
+def get_redis_backend(backend_uri):
+    # From django-redis-cache
+    # This is temporary https://github.com/washort/nuggets/pull/1
+    _, server, params = parse_backend_uri(backend_uri)
+    db = params.pop('db', 1)
+    try:
+        db = int(db)
+    except (ValueError, TypeError):
+        db = 0
+    try:
+        socket_timeout = float(params.pop('socket_timeout'))
+    except (KeyError, ValueError):
+        socket_timeout = None
+    password = params.pop('password', None)
+    if ':' in server:
+        host, port = server.split(':')
+        try:
+            port = int(port)
+        except (ValueError, TypeError):
+            port = 6379
+    else:
+        host = 'localhost'
```

Comment:
server is a combination of hostname:port, so it needs to be split up and passed to the redis connection.